index.js

```diff
@@ -3,12 +3,14 @@ const { exec } = require('@actions/exec')
 const path = require('path')
 
 const run = async () => {
-  const version = core.getInput('version')
-  await exec(path.join(__dirname, 'install-jsonnet.sh'), version)
+  try {
+    const version = core.getInput('version')
+    await exec(path.join(__dirname, 'install-jsonnet.sh'), version)
+  } catch (error) {
+    core.setFailed(`Action failed with error: ${error}`)
+  }
 }
 
-try {
-  run()
-} catch (error) {
+run().catch((error) => {
```

Comment:
Can remove the `.catch` since it's no longer needed :)

Reply:
Oops... Fixed 👍

Reply:
Thanks for doing this. LGTM!